### ISSUE TITLE
Question about \ref{foo} references in mdBook files

### ISSUE THREAD

**Issue by THeK3nger** [rust-lang-nursery/mdBook#656]
Hi everyone,

In the last couple of days I was moving some LaTeX files to a mdbook project. With my great surprise, I noticed that the latex command `\ref{foo}` is a "valid" command. In fact, all my `Algorithm \ref{something}` are rendered as `Algorithm ???`.

This does not look like a CommonMark specification and I cannot find anywhere where such processing is coded in the in the source code mdBook. Moreover, it is not documented anywhere and the correspondant `\label{foo}` command does not work.

I suppose this is some "side-effect" and not something officially supported. If not, there is any documentation for this?

**Comment by Michael-F-Bryan** [comment]
Oops! I think this may have something to do with [MathJax]. Do you have `mathjax-support = true` set in your `book.toml` by any chance?

[MathJax]: https://rust-lang-nursery.github.io/mdBook/format/mathjax.html

**Comment by THeK3nger** [comment]
Yes. I have it. 👍 I am moving into CommonMark parts of papers in LaTeX. 

For your information, the `ref` command is _outside_ of MathJax blocks.
